src/main/java/graphics/GraphicsMain.java

```diff
@@ -107,5 +110,8 @@ public void addDownAlligatorDen(Point2D pos) {
         renderPane.getGame().getBoard().getObjectManager().addDownAlligatorDen(pos);
     }
 
+    public void addChasingElement(Point2D pos){
+        renderPane.getGame().getBoard().getObjectManager().addChasingElement(pos);
```

Comment:
Again, I don't think it's clean to continuously import from bottom to top. We can just do game.addChasingElement(), which will then call all the required methods.

Reply:
Yes. I have said before it is not good Java style to chain the calls, but this embedded test section has all the GraphicsMain-level calls in this style....Is it correct to say that the GameMaker will call Game methods to plant the elements? If so, I'll try to wrap them all into Game-level methods.

Reply:
Yea we’re going to have to change the calls structured in this style to calling game methods. So might as well start now. It’d be nice if you could also do it for the other methods as well.

Reply:
Will do. I'm thinking it may be useful to create a separate GameSeeder class that handles all the element-planting.